CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Added
 
+* patterns can be read in from `stdin` with the `--patterns-from-stdin` and `-z` command line options/flags
```

Comment:
Add reference to the PR to end of line as follows `([#1606](https://github.com/pinterest/ktlint/pull/1606))`